### PR TITLE
loginpagetestcaseaded

### PR DESCRIPTION
**Pull request by Sathish0107**
Login Page Test case added, please review the code

### CODE REVIEW

File: src/pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package pages;
+
+public class LoginPage {
+	
+	public static void main(String[] args) {
+		
+		System.out.println("this is login page");
```
Comment (dkg4484) [inline src/pages/LoginPage.java]: Hey Sathish, Where the assertion ?

Comment (dkg4484) [inline src/pages/LoginPage.java]: Sathish, This is a print statment. Where u developed application?

**Review by dkg4484 — CHANGES_REQUESTED** [review]
Hi Sathish,

I have sent two major changes to be include as part of your code  and then revert the same to me.

Thanks!
Dinesh

**Review by dkg4484 — APPROVED** [review]
Great Job! Your code is correct. 

Its approved.

Please merge your login branch code to master!

**Comment by Sathish0107** [comment]
Merged the code to master

**Pull request merged**